src/devices/GoPiGo3/Sensors/Button.cs

```diff
@@ -0,0 +1,37 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Iot.Device.GoPiGo3.Models;
+
+namespace Iot.Device.GoPiGo3.Sensors
+{
+    /// <summary>
+    /// A Button class
+    /// </summary>
+    public class Button:DigitalInput
+    {
+        /// <summary>
+        /// Button constructor
+        /// </summary>
+        /// <param name="goPiGo">The GoPiGo3 class</param>
+        /// <param name="port">The Groove Port, need to be in the list of SupportedPorts</param>
+        public Button(GoPiGo goPiGo, GroovePort port) : base(goPiGo, port)
+        { }
+
+        /// <summary>
+        /// True if the button is pressed, flase otherwise
+        /// </summary>
+        public bool IsPressed => Value != 0 ? true : false;
+
+        /// <summary>
+        /// Get "Pressed" if the button is pressed "Not pressed" if not
+        /// </summary>
+        public new string ValueAsString => Value != 0 ? "Pressed" : "Not pressed";
```

Comment:
what about implementing `ToString()` instead? This way you can just `Console.WriteLine($"state: {button}")`

Reply:
Yes, now I got it. Will replace all the ValueAsString as ToString()